content/en/developers/faq/is-there-an-alternative-to-dogstatsd-and-the-api-to-submit-metrics-threadstats.md

```diff
@@ -32,11 +37,29 @@ Threadstats is good for you if...
 2. You don't want to install the Datadog Agent
 3. Or you want more flexibility than DogStatsD has to offer; for instance, you want to send metrics with timestamps
 
-## References
+## Threadstats variations with DogStatsD
+
+As in [DogStatsD](/developers/dogstatsd), Threadstats performs data aggregation for performance reasons.
+
+### Variations
+
+* The main difference is that metrics received by Threadstats may already have a timestamp
+* Metrics are not aggregated via a centralized server, but they are aggregated and flushed in a Python thread of your script. So you'll get a per-script aggregation instead of a per host aggregation
+
+To handle timestamps, Threadstats uses 2 parameters: a flush interval and a roll-up interval.
 
-Documentation: http://datadogpy.readthedocs.org/en/latest/#datadog-threadstats-module
+* The flush interval defines the time interval between two consecutive {data aggregation + data submission}.
```

Comment:
Unsure what is being said here. Is it between two consecutive data aggregation-submission instances?

Reply:
indeed, prior to be sent the data goes through 2 phases: 1. aggregation  then 2. submission. The flush interval is the sum of those two phases.

Reply:
Let's change it to 
```suggestion
* The flush interval defines the time interval between two consecutive data aggregation-and-submission events.
```